src/applications/static-pages/sass/modules/_m-facilities.scss

```diff
@@ -40,5 +40,162 @@ $duo-tone-lighten: #000000;
 }
 
 .facility-satisfaction-tile {
-  width: 147px;
+  width: 50%;
+
+  @media all and (min-width: 768px) {
```

Comment:
We have helpers we normally use for breakpoints: `@include media($large-screen) {` and breakpoint variables here: https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/master/packages/formation/sass/base/_b-breakpoints.scss#L21

Wherever we still need these breakpoints (see my other comments), can we switch to using that helper and the breakpoint variables?